src/Draco.Lsp/Server/Workspace/IDidDeleteFiles.cs

```diff
@@ -0,0 +1,15 @@
+using System.Threading;
+using System.Threading.Tasks;
+using Draco.Lsp.Attributes;
+using Draco.Lsp.Model;
+
+namespace Draco.Lsp.Server.Workspace;
+
+public interface IDidDeleteFiles
+{
+    [RegistrationOptions("workspace/didDeleteFiles")]
+    public FileOperationRegistrationOptions FileOperationRegistrationOptions { get; }
```

Comment:
Rename please, this will collide with other file operations once we introduce them. Make it something like `DidDeleteFileRegistrationOptions`